Bonus.js

```diff
@@ -0,0 +1,30 @@
+
+// Removes all children and re-add the inventory
+function refreshList( _parentId ){
+    const children = document.getElementById( _parentId ).children;
+    for ( let i = children.length-1; i >= 0; i-- )
+        children[i].remove();
+
+    listAllTvs( inventory, _parentId );
+}
+
+// sort by sold
+const bt_sold = document.getElementById('sort-sold');
+bt_sold.addEventListener('click', () => {
+    inventory.sort( ( a, b) => a.sold - b.sold);
+    refreshList("tv-content-all");
+} );
+
+// sort by price
+const bt_price = document.getElementById('sort-price');
+bt_price.addEventListener('click', () => {
+    inventory.sort( ( a, b) => a.price - b.price);
+    refreshList("tv-content-all");
+} );
+
+// sort by ambiLight
+const bt_ambiLight = document.getElementById('sort-ambilight');
+bt_ambiLight.addEventListener( 'click', () => {
+    inventory.sort( ( a, b) => b.options.ambiLight ? 1 : -1);
```

Comment:
Dit kun je versimpelen naar `inventory.sort((a, b) => a.price - b.price);`

Reply:
Klopt deze feedback? Het lijkt mij dat het dan op price gesorteerd wordt ipv amibLight. Op regel 21 heb ik al wel deze sort gebruikt.

Reply:
Je hebt helemaal gelijk, ik bedoelde:
`inventory.sort((a, b) => b.options.ambiLight);`

Reply:
Hmm... deze oplossing lijkt niet te werken of was het toch nog iets anders dan wat je nu hebt gezet?